File: tools/pas2php-code-6/pas2php_include/p2psysutils.php
<?PHP
// 
// Converted From Pascal By Pas2Php v0.8
// Converts Object Pascal to PHP
// http://www.wascal.net/?page=pas2php
// Pas2Php v0.8 is Copyright (C) 2014-2016 Derek John Evans
// 
include_once('system.php');
include_once('sysutils.php');
/*
function AnsiCompare(const A: string;const B: string;
                    const ACaseSensitive: Boolean) : Integer;
*/
function AnsiCompare($A, $B, $ACaseSensitive) {
  // Result : Integer;
  $Result = 0;
  if ($ACaseSensitive) {
    $Result = AnsiCompareStr($A, $B);
  }
  else {
    $Result = AnsiCompareText($A, $B);
  }
  return $Result;
}
/*
function AnsiSame(const A: string;const B: string;
                 const ACaseSensitive: Boolean) : Boolean;
*/
function AnsiSame($A, $B, $ACaseSensitive) {
  // Result : Boolean;
  $Result = false;
  if ($ACaseSensitive) {
    $Result = AnsiSameStr($A, $B);
  }
  else {
    $Result = AnsiSameText($A, $B);
  }
  return $Result;
}
?>

File: tools/pas2php-code-6/pas2php_include/p2psysutils.pas
{
 ***************************************************************************
 *                                                                         *
 *   This source is free software; you can redistribute it and/or modify   *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 2 of the License, or     *
 *   (at your option) any later version.                                   *
 *                                                                         *
 *   This code is distributed in the hope that it will be useful, but      *
 *   WITHOUT ANY WARRANTY; without even the implied warranty of            *
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU     *
 *   General Public License for more details.                              *
 *                                                                         *
 *   A copy of the GNU General Public License is available on the World    *
 *   Wide Web at <http://www.gnu.org/copyleft/gpl.html>. You can also      *
 *   obtain it by writing to the Free Software Foundation,                 *
 *   Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.        *
 *                                                                         *
 ***************************************************************************
}

unit P2PSysUtils;

interface

uses
  SysUtils;

function AnsiCompare(const A, B: string; const ACaseSensitive: boolean): integer;
function AnsiSame(const A, B: string; const ACaseSensitive: boolean): boolean;

implementation

function AnsiCompare(const A, B: string; const ACaseSensitive: boolean): integer;
begin
  if ACaseSensitive then begin
    Result := AnsiCompareStr(A, B);
  end else begin
    Result := AnsiCompareText(A, B);
  end;
end;

function AnsiSame(const A, B: string; const ACaseSensitive: boolean): boolean;
begin
  if ACaseSensitive then begin
    Result := AnsiSameStr(A, B);
  end else begin
    Result := AnsiSameText(A, B);
  end;
end;

end.
